astropy/table/serialize.py

```diff
@@ -58,7 +58,6 @@ def _represent_mixin_as_column(col, name, new_cols, mixin_cols,
     ``MaskedColumn`` can also be serialized.
     """
     obj_attrs = col.info._represent_as_dict()
-    ordered_keys = col.info._represent_as_dict_attrs
```

Comment:
I removed this thinking order was no longer an issue, since `dict` are guaranteed to be ordered - but I'm not sure this is true for `set` - so perhaps this is too much. If so, it'll need some thought.

Reply:
I don't recall seeing any statements about set order being preserved.

Reply:
Actually, I think I managed to get myself confused: we call `_represent_as_dict()` here, which is these days guaranteed to be ordered (as it is done in order of `_represent_as_dict_attrs`).  So, that explains why no tests failed because of this!